src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double sideA;
+    private double sideB;
+    private double sideC;
+
+    public RightTriangle(double sideA, double sideB, double sideC, String color) {
+        super(color);
+        this.setSideA(sideA);
+        this.setSideB(sideB);
+        this.setSideC(sideC);
+    }
+
+    @Override
+    public double area() {
+        double s;
```

Comment:
You can directly return the calculated area without using an extra variable, like this: return (getSideA() * getSideB()) / 2;